packages/edit-site/src/components/header/undo-redo/redo.js

```diff
@@ -1,18 +1,19 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, isRTL } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
-import { redo as redoIcon } from '@wordpress/icons';
+import { redo as redoIcon, undo } from '@wordpress/icons';
 import { displayShortcut } from '@wordpress/keycodes';
+import { store as coreStore } from '@wordpress/core-data';
 
 export default function RedoButton() {
-	const hasRedo = useSelect( ( select ) => select( 'core' ).hasRedo() );
-	const { redo } = useDispatch( 'core' );
+	const hasRedo = useSelect( ( select ) => select( coreStore ).hasRedo() );
+	const { redo } = useDispatch( coreStore );
 	return (
 		<Button
-			icon={ redoIcon }
+			icon={ ! isRTL() ? redoIcon : undo }
```

Comment:
It's super confusing so maybe it should have a comment in all places? It's a question. I don't know 😄 

Should it be `redoIcon` and `undoIcon` for consistency? I'm not sure why the first is aliased though.

Reply:
I aliased all icons for better readability. It was aliased due to the dispatch with the same name.